Detectors/TOF/prototyping/drawTOFgeometry.C

```diff
@@ -0,0 +1,105 @@
+#if (!defined(__CINT__) && !defined(__CLING__)) || defined(__MAKECINT__)
```

Comment:
Hi, I'm sorry for this. I started working from some already present macro i.e. Detectors/Passive/macro/PutFrameInTop.C which did not have the statement so I forgot. Adding it straight away! Should I open a new PR or include the commit in a future one?